### PR TITLE
BUG: Use consistent ITK_USE_CLANG_FORMAT variable name

### PR DESCRIPTION
**Pull request by thewtex**
Across ITKClangFormatSetup.cmake and ITKModuleClangFormat.cmake.

### CODE REVIEW

**Review by dzenanz — APPROVED** [review]

**Review by jhlegarreta — APPROVED** [review]

**Comment by jhlegarreta** [comment]
Sorry for my late thought: now I'm wondering whether there will be some clash across the *two* variables, since they are named the same, but look as if they are defined in two different places. Shouldn't the variable be defined at a single place and be re-used by the other file?

**Comment by thewtex** [comment]
They are in two places so the one file can be used independently from the other, e.g. ITK versus ITK external module build.

File: CMake/ITKClangFormatSetup.cmake [hunk 1]
```diff
@@ -55,7 +55,7 @@ endif()
 if(CLANGFORMAT_EXECUTABLE)
   mark_as_advanced(CLANGFORMAT_EXECUTABLE)
 endif()
-if ( ITK_USE_CLANGFORMAT AND NOT EXISTS "${CLANGFORMAT_EXECUTABLE}")
+if ( ITK_USE_CLANG_FORMAT AND NOT EXISTS "${CLANGFORMAT_EXECUTABLE}")
```
Comment (N-Dekker) [inline CMake/ITKClangFormatSetup.cmake:58]: `CLANGFORMAT_EXECUTABLE`? Or `CLANG_FORMAT_EXECUTABLE`? 😺

Comment (thewtex) [reply CMake/ITKClangFormatSetup.cmake:58]: We could use `CLANG_FORMAT` consistently.

**Review by dzenanz — APPROVED** [review]

**Review by hjmjohnson — APPROVED** [review]

**Pull request merged**